### PR TITLE
RELEASE 2.2.9: Add missing dependency

### PR DESCRIPTION
**Pull request by paulmelnikow**
Caused a build failure: https://circleci.com/gh/bodylabs/baiji-pod/3

### CODE REVIEW

File: baiji/util/test_reachability.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import unittest
+
+class TestReachability(unittest.TestCase):
+    def test_internet_reachable(self):
+        # We assume these tests run with a reachable Internet. This test is
```
Comment (dlsmith) [inline baiji/util/test_reachability.py]: Not all of the tests should require access to the internet though. Most should be able to run by mocking key boto functionality, no? Integration tests would require access to the internet, but I think a test case mixin would be a more appropriate place for this check.

I'm not suggesting you remove the test, just that the spirit of the comment might be changed.

**Comment by dlsmith** [comment]
👍

File: baiji/util/test_reachability.py (hunk 1 above)
Comment (paulmelnikow) [inline baiji/util/test_reachability.py]: Currently, as a group, the "unit tests" in this repo are integration tests which require S3. What you're suggesting is possible.

Will reword the comment to that effect.

**Comment by paulmelnikow** [comment]
Thanks!

**Pull request merged**